src/main/java/uk/gov/hmcts/sscs/controller/NotificationController.java

```diff
@@ -34,7 +52,45 @@ public void sendNotification(
         LOG.info("Ccd Response received for case id: {}", ccdResponseWrapper.getNewCcdResponse().getCaseId());
 
         authorisationService.authorise(serviceAuthHeader);
-        service.createAndSendNotification(ccdResponseWrapper);
+        service.createAndSendNotification(new CcdNotificationWrapper(ccdResponseWrapper));
+    }
+
+    @RequestMapping(value = "/coh-send", method = POST, produces = APPLICATION_JSON_VALUE)
+    public void sendCohNotification(
+            @RequestHeader(AuthorisationService.SERVICE_AUTHORISATION_HEADER) String serviceAuthHeader,
+            @RequestBody CohEvent cohEvent) {
+        String caseId = cohEvent.getCaseId();
+        LOG.info("Coh Response received for case id: {}", caseId);
+
+        authorisationService.authorise(serviceAuthHeader);
```

Comment:
Do you think it would be better if this IDAM stuff was refactored into it's own class which can be reused by any other classes wanting to use IDAM?